src/main/java/org/storageart/storageart/entities/User.java

```diff
@@ -1,23 +1,54 @@
 package org.storageart.storageart.entities;
 
-import javax.persistence.*;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
 
-import java.util.Optional;
+import javax.persistence.*;
+import java.util.Collection;
+import java.util.Iterator;
 
 @Entity
 @Table(name="users")
-public class User {
+public class User implements UserDetails {
```

Comment:
It's not a good idea to return entity object to view. Read about DTO (data transfer object), and use this approach  here